arcgis-ios-sdk-samples/Layers/Add ENC exchange set/AddENCExchangeSetViewController.swift

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class AddENCExchangeSetViewController: UIViewController {
+    /// The map view managed by the view controller.
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap(basemap: .oceans())
+        }
+    }
+    
+    /// Load the ENC dataset and add to the map view.
+    ///
+    /// - Parameter mapView: The map view managed by the view controller.
+    func addENCExchangeSet(mapView: AGSMapView) {
+        guard let map = mapView.map else { return }
+        // Load catalog file in ENC exchange set from bundle.
+        let catalogURL = Bundle.main.url(
+            forResource: "CATALOG",
+            withExtension: "031",
+            subdirectory: "ExchangeSetwithoutUpdates/ExchangeSetwithoutUpdates/ENC_ROOT"
+        )!
+        let encExchangeSet = AGSENCExchangeSet(fileURLs: [catalogURL])
+        
+        // URL to the "hydrography" data folder that contains the "S57DataDictionary.xml" file.
+        let hydrographyDirectory = Bundle.main.url(
+            forResource: "S57DataDictionary",
+            withExtension: "xml",
+            subdirectory: "hydrography/hydrography"
```

Comment:
Same with `hydrography`. Any way we could update the zip to remove one of the `hydrography` directories?

Reply:
After discussion with team, the current solution is to create replicas of these portal items with correct folder structure, instead of fixing the original files. But I'm afraid that will add burdens to the data management in the long run. (People may get confused why there are items with same content but different IDs.) 

@mikewilburn what do you think?

Reply:
I'm not sure of the history that brought about the seemingly redundant `/ExchangeSetwithoutUpdates/ExchangeSetwithoutUpdates/` and `hydrograpy/hydrography` directory structure.

This is data we didn't produce ourselves, correct? It's supplementary data created by Esri/Runtime to permit the API to function as expected?

Reply:
These 2 items are replaced by replicas with folder structure fixed. However, the links in the README is still pointing to the original items to avoid confusion.